Python-packages/covidcast-py/shapefiles/county/cb_2019_us_county_5m.cpg

```diff
@@ -0,0 +1 @@
+UTF-8
```

Comment:
Did these shapefiles come from the Census? (Maybe the ones we already used in the R package?) Would be good to have a note somewhere with the source, for future updating and for credit. And in case there are licensing restrictions. (Though Census products would all be public domain)

Reply:
Yea they're from here https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html. Added a note to the plotting docstring.